hbase-server/src/main/java/org/apache/hadoop/hbase/regionserver/HStore.java

```diff
@@ -2529,6 +2529,8 @@ protected OffPeakHours getOffPeakHours() {
   public void onConfigurationChange(Configuration conf) {
     this.conf = new CompoundConfiguration()
             .add(conf)
+            .addBytesMap(region.getTableDescriptor().getValues())
```

Comment:
Better abstract a util method to do the merging of the configurations and call it from both the constructor and here?

Reply:
Mind addressing this issue?

When implementing HBASE-26067, we also need to use something like this when we want to create a store file tracker at master side. So if you can provide a util class, it could also benefit the HBASE-26067 branch.

@mymeiyi

Reply:
Mind addressing this issue?

When implementing HBASE-26067, we also need to use something like this when we want to create a store file tracker at master side. So if you can provide a util class, it could also benefit the HBASE-26067 branch.

@mymeiyi

Reply:
Can you review https://github.com/apache/hbase/pull/3671 firstly, then let me fix this and UT, thanks.

Reply:
Ah, OK, let me take a look...